### PR TITLE
feat: Add bitbucket support

### PR DESCRIPTION
**Pull request by pabelanger**
This change adds support for bitbucket pull request. This is heavily
insprired by the work of Tristan Cacqueray <tdecacqu@redhat.com>.

Thanks Tristan!

Signed-off-by: Paul Belanger <pabelanger@gmail.com>

### CODE REVIEW

**Comment by pabelanger** [comment]
@jd @TristanCacqueray Let me know what you think, I'll push up some documentation shortly. I've been using git pull-request with github for years, would be nice to also use it with bitbucket.org.

**Comment by pabelanger** [comment]
okay, should be ready now.

**Comment by pabelanger** [comment]
@sileht including you too

**Comment by pabelanger** [comment]
@sileht including you too

File: git_pull_request/bitbucket.py [hunk 1]
```diff
@@ -0,0 +1,196 @@
...
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+# implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import daiquiri
+import requests
+
+
+LOG = daiquiri.getLogger("git-pull-request")
```
Comment (jd) [inline git_pull_request/bitbucket.py:18]:
```suggestion
LOG = daiquiri.getLogger(__name__)
```
Maybe? Not sure if there's a downside here.

**Review by jd — APPROVED** [review]
LGTM.

**Pull request merged**